packages/create-next-app/templates/default-tw/ts/pages/index.tsx

```diff
@@ -62,7 +62,7 @@ export default function Home() {
           </a>
         </div>
       </main>
-      <footer className="row-start-3 flex gap-6 flex-wrap items-center justify-center">
+      <footer className="row-start-3 flex flex-wrap items-center justify-center gap-6">
```

Comment:
What are these changes about? They seem unrelated and should therefore be file in a separate PR.

Reply:
Not sure, they got updated when I ran the linter.

Reply:
Let's revert these then.

Reply:
Reverted :D